pkg/client/mount.go

```diff
@@ -131,10 +132,11 @@ func (v *Volume) ExpandFs(path, fs string, size int64) error {
 	if err != nil {
 		return err
 	}
+        co.Debugf(ctxt, "Expand to size requested = %d", size * units.GiB)
 	if err := checkDeviceSize(ctxt, device, size); err != nil {
 		return err
 	}
-	return expandFs(ctxt, path, fs)
+        return expandFs(ctxt, device, fs)
```

Comment:
your tabs are off (see above two other occurences too))

Reply:
Not sure why GitHub shows 2 spaces off. In the code file, I see 8 spaces of indentation across all functions. Checked both master and dev branches and they both show 8 spaces of indentation.

Reply:
Fixed.